application/app/src/test/resources/application.yaml

```diff
@@ -0,0 +1,13 @@
+server:
+  port: 9095
+
+api:
+  endpoint: http://localhost:9095/dosapi/dosservices/v0.0.1/services/byfuzzysearch
```

Comment:
Can we get that information from a profile (make variables) rather than hardcoding them in the `yaml` file?